code/components/citizen-server-impl/include/state/ServerGameState.h

```diff
@@ -383,6 +383,7 @@ struct SyncEntityState
 	uint64_t lastFrameIndex;
 	uint16_t uniqifier;
 	uint32_t creationToken;
+	float overrideCullingRadius;
```

Comment:
`= 0.0f;`. Without any default set either here or in the constructor it'll be uninitialized memory, which could be _anything_!